src/main/java/objectsBasicsMandatoryPracticalTasks/Clock/Clock.java

```diff
@@ -0,0 +1,137 @@
+package objectsBasicsMandatoryPracticalTasks.Clock;
+
+class Clock {
+    private long hours;
+    private long minutes;
+    private long seconds;
+    private static final int SEC_IN_DAYS = 86400;
+    private static final int SEC_IN_HOURS = 3600;
+    private static final int SEC_IN_MINUTE = 60;
+
+    Clock() {
+        this.hours = 12;
+        this.minutes = 0;
+        this.seconds = 0;
+    }
+
+    Clock(long hours, long minutes, long seconds) {
```

Comment:
No validation is applied in constructor, so that I can specify 25 for hours for example or 1235 minutes and it seems these values won't be valid